README.md

```diff
@@ -25,7 +25,13 @@ Modernizr tests which native CSS3 and HTML5 features are available in the curren
   
   - `class` to `es6class` to keep in line with the rest of the es-tests
 
-- These tests got removed:
+- Following tests got moved in subdirectories:
+
+  - `cookies`, `indexeddb`, `indexeddbblock`, `quota-management-api`, `userdata` moved into the storage subdirectory
```

Comment:
Seems like autocorrect has autowrong in indexedblob 😅. Just noticed.

Reply:
Good catch!